mantidimaging/gui/widgets/spectrum_widgets/fitting_param_form_widget.py

```diff
@@ -43,6 +46,21 @@ def __init__(self, presenter: SpectrumViewerWindowPresenter, parent=None) -> Non
         self.run_fit_button.clicked.connect(self.presenter.run_region_fit)
         self._param_values_logged = False
 
+        self.chi2_label = QLabel("Fitting Quality (χ²): N/A")
+        main_layout.addWidget(self.chi2_label)
+        self.metric_picker = QComboBox(self)
```

Comment:
It would be simpler to remove the choice and just display 2 values. E.g.

RSS = 0.25
RSS/DoF = 0.11